pkg/cli/debug/debug.go

```diff
@@ -453,20 +453,14 @@ func (o *DebugOptions) RunDebug() error {
 			}
 			return fmt.Errorf(msg)
 			// switch to logging output
-		case err == krun.ErrPodCompleted, err == conditions.ErrContainerTerminated, !o.Attach.Stdin:
```

Comment:
My proposal for this switch:
```go
case err == krun.ErrPodCompleted, err == conditions.ErrContainerTerminated, !o.Attach.Stdin:
	return o.logsOptions(pod)
case err == conditions.ErrNonZeroExitCode:
	o.logsOptions(pod)
	return conditions.ErrNonZeroExitCode
case err != nil:
	return err
case !o.Attach.Stdin:
	return o.logsOptions(pod)
```
oh, and please change `logsOptions` to `getLogs` or something like that.

Reply:
updated - but handled the error from geLogs(pod) - or should I not? :
```
case err == conditions.ErrNonZeroExitCode:
	if err = o.getLogs(pod); err != nil {
            return err
         }
	return conditions.ErrNonZeroExitCode
```

Reply:
Perfect :+1: